manual/first_steps_with_root/index.md

```diff
@@ -168,10 +79,10 @@ For more information on Cling, → see [Cling]({{ '/cling' | relative_url }}).
 
 > **Note**
 >
-> When using the interactive interpreter Cling, unlike to pure C++ language, no semicolon
-> (`;`) is required at the end of the line.
+> Cling provides a user experience that is closer to a typical interpreter, e.g. [IPython](https://ipython.org/).
+> Therefore, unlike pure C++ language, no semicolon (`;`) is required at the end of the line.
 
-### Simple commands
+#### Simple commands
```

Comment:
Simple operation should be:
```
{% highlight C++ %}
   root [0] 21+21
   (int) 42
{% endhighlight %}

{% highlight C++ %}
   root [1] sqrt(42)
  (double) 6.4807407
{% endhighlight %}

Relational operators:
{% highlight C++ %}
   root [2] 42 > 98
   (bool) false
{% endhighlight %}

Calling a function from a ROOT class like `TMath::Pi`:
{% highlight C++ %}
   root [3] TMath::Pi()
   (double) 3.1415927
{% endhighlight %}

```
The output do not correspond to the reality and Math::Pi() should TMath::Pi() (T missing)

Reply:
Done thanks!